File: android/src/main/java/nj/com/flutter_scan_view/FlutterScanViewPlugin.java
package nj.com.flutter_scan_view;

import io.flutter.plugin.common.PluginRegistry;

public class FlutterScanViewPlugin {
    public static void registerWith(PluginRegistry.Registrar registrar) {
        registrar.platformViewRegistry().registerViewFactory("plugins.flutter.io/njscanview", new NJScanViewFactory(registrar.messenger()));

    }
}

File: android/src/main/kotlin/nj/com/flutter_scan_view/FlutterScanViewPlugin.kt
package nj.com.flutter_scan_view


import io.flutter.plugin.common.PluginRegistry.Registrar

class FlutterScanViewPlugin {
    companion object {
        @JvmStatic
        fun registerWith(registrar: Registrar) {
            val scanViewFactory = NJScanViewFactory(registrar.messenger());
            registrar.platformViewRegistry().registerViewFactory("plugins.flutter.io/njscanview", scanViewFactory);
        }
    }


}
